templates/ros2_cpp_pkg/{{ package_name }}/src/{{ node_name }}.cpp.jinja

```diff
@@ -126,6 +126,7 @@ void {{ node_class_name }}::declareAndLoadParameter(const std::string& name,
         for (const auto& element : param) ss << element << (&element != &param.back() ? ", " : "]");
       } else {
         ss << param;
+        this->set_parameters({rclcpp::Parameter(name, param)});
       }
       RCLCPP_WARN_STREAM(this->get_logger(), ss.str());
```

Comment:
Should be done in the else-clause of `if (is_required)`, not only for non-vectors, right?

```suggestion
      }
      RCLCPP_WARN_STREAM(this->get_logger(), ss.str());
      this->set_parameters({rclcpp::Parameter(name, param)});
```

Reply:
Thanks close needed to modify to  `this->set_parameters({rclcpp::Parameter(name, rclcpp::ParameterValue(param))});` to work with vectors